architecture/crypto.rst

```diff
@@ -67,11 +67,11 @@ default implementation, mostly based on LibTomCrypt_, is as follows:
 
 .. code-block:: c
     :caption: File: core/crypto/crypto.c
-    
+
     /*
      * Default implementation for all functions in crypto.h
      */
-    
+
```

Comment:
It seems these non empty blank indentation lines are needed for the rendered tool to consider  the block as a single code-bock. Please cross-check whether it's ok or not not have these changes.

Reply:
Ok. Rendrering on latexpdf and *html also look ok on my side.
This changes are unrelated to the commit description. @ldts, could you move these to a specific commit?